### PR TITLE
Update dead link

### PR DESCRIPTION
**Pull request by kiplingi**
### 1. Why is this change necessary?
A link was dead and ist now pointing to the updated location.

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [x] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

<a href="https://gitpod.io/#https://github.com/shopware/platform/pull/2831"></a>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ If you want more details about available licensing or the contribution agreement
 
 ## Contributing to the Shopware code base
 If you want to learn how to contribute code to Shopware, please refer to [Contributing Code](https://docs.shopware.com/en/shopware-platform-dev-en/contribution/contributing-code?category=shopware-platform-dev-en/contribution).
-Also make sure that you add a changelog file which describes your changes in a meaningful way. For more information refer to [this document](https://github.com/shopware/platform/blob/master/adr/2020-08-03-Implement-New-Changelog.md#workflow).
+Also make sure that you add a changelog file which describes your changes in a meaningful way. For more information refer to [this document](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md).
```
Comment (trandangtri) [inline CONTRIBUTING.md:9]: Thanks for good catch, @kiplingi 

I'm rather greedy :) We have similar one in `src/Docs/Resources/current/70-contribution/10-contribution-guideline.md`. You don't mind if you help to fix it as well in a single MR, then we could provide a proper documentation for people?

Thanks in advance.

**Review by trandangtri — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (mitelg) [reply CONTRIBUTING.md:9]: isn't the `src/Docs` deprecated anyway?

Comment (shyim) [reply CONTRIBUTING.md:9]: Yes you can ignore it

Comment (kiplingi) [reply CONTRIBUTING.md:9]: Sounds mysterious ;) Is the doc moving, or do we get a new one?

Comment (mitelg) [reply CONTRIBUTING.md:9]: the docs are located here for a while now :slightly_smiling_face: https://github.com/shopware/docs

**Comment by keulinho** [comment]
Hey @kiplingi thanks again for your contribution, your PR got merged! :tada: :blue_heart:

**Pull request closed without merging**